trustwallet/wallet-core
# [Sec]: Zeroize the memory for each `TWString`

## Description

Zeroize the memory for each `TWString` and for the mnemonic data.

## How to test

Run C++, iOS, Android tests.
Use memory dump tools to make sure if the mnemonic does not present in the memory.

## Types of changes

## Checklist




If you're adding a new blockchain
